modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -8,17 +8,146 @@ label:AuraDB-Business-Critical[]
+* *Org SSO:* Allows org admins to restrict how users log in when they are trying to access the org.
 Access beyond log-in is managed via RBAC.
 
-* *Tenant level SSO:*  Impacts new database instances created within that tenant. 
-It ensures users logging in with SSO have access to the database instances within the tenant. 
-It depends on RBAC if the user can access and view or modify data within the database instances themselves. 
+* *Project-level SSO:*  Impacts new database instances created within that project.
+It ensures users logging in with SSO have access to the database instances within the project.
+It depends on RBAC if the user can access and view or modify data within the database instances themselves.
 For this level, the role mapping may be used to grant users different levels of access based on a role in their identity provider (IdP).
-It *does not* give access to edit the tenant settings, for example to edit the tenant name, network access, or to edit the instance settings such as to rename an instance, or pause and resume.
+It *does not* give access to edit the project settings, for example to edit the project name, network access, or to edit the instance settings such as to rename an instance, or pause and resume.
+
+=== SSO Org level roles
+
+The following roles are available at the org level and these are assigned via invitation:
+
+* Owner
+* Admin
+* Member
+
+:check-mark: icon:check[]
+
+.Roles
+[opts="header",cols="3,1,1,1"]
+|===
+| Capability
+| Owner
+| Admin
+| Member
+
+| List org
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| List projects
+| {check-mark}
+| {check-mark}
+| {check-mark}
+
+| Update org
+| {check-mark}
+| {check-mark}
+|
+
+| Add projects
+| {check-mark}
+| {check-mark}
+|
+
+| List existing SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Add SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| List project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Update project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Delete project-level SSO configs
+| {check-mark}
+| {check-mark}
+|
+
+| Invite non-owner users to org
+| {check-mark}
+| {check-mark}
+|
+
+| List users
+| {check-mark}
+| {check-mark}
+|
+
+| List roles
+| {check-mark}
+| {check-mark}
+|
+
+| List members of a project
+| {check-mark}
+| {check-mark} footnote:[An admin can only list members of projects the admin is also a member of.]
+|
+
+// | Add customer information for a trial within org
+// | {check-mark}
+// | {check-mark}
+// |
+
+// | List customer information for a trial within org
+// | {check-mark}
+// | {check-mark}
+// |
+
+// | List seamless login for org
+// | {check-mark}
+// | {check-mark}
+// |
+
+// | Update seamless login for org
+// | {check-mark}
+// | {check-mark}
+// |
+
+| Invite owners to org
+| {check-mark}
+|
+|
+
+| Add owner
+| {check-mark}
+|
+|
+
+| Delete owners
+| {check-mark}
+|
+|
+
+| Transfer projects
+| {check-mark}
+|
+|
```

Comment:
This is a duplicate of the `Transfer projects to and from the org` right? they are the same in the code for what they can do. You cant' transfer a project within an org cause the project is already a part of the org so there's nothing to do in that case.